scopes/html/html-docs-app.ts

```diff
@@ -0,0 +1,26 @@
+import { htmlToReact } from './html-to-react';
+import mapObject from 'map-obj'
+// maybe extract React/docs to its own component?
+// @ts-ignore TODO - fix
+import DocsRoot from '@teambit/react/dist/docs/index.js';
```

Comment:
do not access internal paths of react. never.
if you need, you can export if from the react aspect index file

Reply:
Yeah, this was just for a POC to see if we can get compositions working in the Overview
@KutnerUri we need to now work out how to get this working without the hack

Reply:
@benjgil - just move docs from `scopes/react/react/docs` to `scopes/react/ui/docs-app`, add add it as `teambit.react/ui/docs-app`

then both `react.env` and `html-docs-app.ts` can use it.

make sure to update this at react.env:
```
getDocsTemplate() {
  return require.resolve('@teambit/react.ui.docs-app');
}
```

and test that it works.